lib/ansible/modules/web_infrastructure/jenkins_plugin.py

```diff
@@ -448,31 +450,39 @@ def install(self):
                         self.params['name'],
                         self.params['version']))
 
+            sha1sum_old = base64.b64encode(sha1_old.digest())
             if (
                     self.params['updates_expiration'] == 0 or
                     self.params['version'] not in [None, 'latest'] or
-                    md5sum_old is None):
+                    sha1sum_old is None):
 
                 # Download the plugin file directly
                 r = self._download_plugin(plugin_url)
 
                 # Write downloaded plugin into file if checksums don't match
-                if md5sum_old is None:
+                if sha1sum_old is None:
```

Comment:
There doesn't appear to be any way for `sha1sum_old` to be `None`.

Reply:
this seems like it should have been `sha1_old` and not `sha1sum_old`